### PR TITLE
Fixed issue #3343: Scroll bottom button misbehaviour

### LINKED ISSUE #3343: Scroll up button misbehaves
**Actual Behaviour**
Screenshot from 2020-01-20 13-32-26

**Expected Behaviour**

Scroll up button should be hidden in such cases

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by elit-altum**
Fixes #3343 

Changes: 
- Changed the else-if block display conditions for the scroll bottom button to appear and disappear which were controlled by component state.

Demo Link : https://pr-3422-fossasia-susi-web-chat.surge.sh/

ScrollBottomNotWorkingFixed

### CODE REVIEW

File: src/components/shared/ScrollTopButton.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const ScrollTopFab = styled(Fab)`
   top: ${props => props.height - 95 + 'px'};
   right: ${props => (props.width < 1100 ? '70px' : '100px')};
   margin: 15px;
-  z-index: 9000;
+  z-index: 2;
```
Comment (snitin315) [inline src/components/shared/ScrollTopButton.js]: This change in Z-Index is not required.

Comment (elit-altum) [reply src/components/shared/ScrollTopButton.js]: This is required as the scroll to top button which leads to the top of the webpage conflicts with the chat box input. See here,
ScrollTopOverride

Comment (elit-altum) [reply src/components/shared/ScrollTopButton.js]: The z-index puts the chat modal over the button to resolve this issue.

File: src/components/ChatApp/MessageSection/MessageSection.react.js [hunk 2]
```diff
@@ -395,17 +395,21 @@ class MessageSection extends Component {
...
         this.setState({
-          showScrollBottom: true,
+          showScrollBottom: false,
           showScrollTop: true,
         });
       } else if (
-        scrollValues.scrollHeight - Math.ceil(scrollValues.scrollTop) ===
-        scrollValues.clientHeight
+        !(showScrollBottom && showScrollTop) &&
+        scrollValues.clientHeight + scrollValues.scrollTop <
```
Comment (maze-runnar) [inline src/components/ChatApp/MessageSection/MessageSection.react.js]: add Math.ceil(scrollVakues.scrollTop) .

Comment (elit-altum) [reply src/components/ChatApp/MessageSection/MessageSection.react.js]: @maze-runnar Done!

**Comment by elit-altum** [comment]
> 
> 
> @elit-altum resolve conflicts

Done!

**Comment by elit-altum** [comment]
> 
> 
> Conflicts are not resolved yet.

Conflicts resolved

**Pull request merged**